src/lib/constants/index.js

```diff
@@ -56,3 +56,5 @@ export const MEDIATYPE_DOCKER_MANIFEST_V2 =
 export const RELEASE_NAME_PATTERN = /^(?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63}$/g;
 export const RELEASE_NAME_AIRFLOW_PATTERN = /^\/((?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63})\/(airflow|flower)/;
 export const VALID_DOCKER_IMAGE_NAME = /^(?<releaseName>(?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63})\/airflow$/;
+
+export const REGISTRY_EVENT_UPDATED = "v1.registry.event.updated";
```

Comment:
```suggestion
export const REGISTRY_EVENT_UPDATED = "houston.deployment.image.updated";
```

@adam2k what do you think about this to keep consistency with the other names here https://github.com/astronomer/houston-api/pull/312/files#diff-e21e41780a16c3368512192a60551ee6R60-R63